mast_aladin/app_sidecar.py

```diff
@@ -146,9 +158,28 @@ def _resolve_apps(self, apps, include_aladin, include_jdaviz, use_current_apps):
                 "run `pip install jdaviz`.",
                 UserWarning
             )
-        return apps
 
-    def _attach_sidecars(self, apps, anchor, title):
+        global jdaviz_counter, aladin_counter, other_counter
```

Comment:
the `AppSidecarManager` is intended to only be initialized once by the user, and all instance of sidecar managed through that instance. With that in mind, these should be class variables on `AppSidecarManager` instead of global variables